### PR TITLE
update comment string in ui.py

### PR DESCRIPTION
**Pull request by hamogu**

### CODE REVIEW

File: astropy/io/ascii/ui.py [hunk 1]
```diff
@@ -103,12 +103,12 @@ def read(table, guess=None, **kwargs):
...
     :param names: list of names corresponding to each data column
-    :param include_names: list of names to include in output (default=None selects all names)
+    :param include_names: list of names to include in output (default=``None`` selects all names)
     :param exclude_names: list of names to exlude from output (applied after ``include_names``)
-    :param fill_values: specification of fill values for bad or missing table values (default=('', '0'))
+    :param fill_values: specification of fill values for bad or missing table values (default= ``('', '0')``)
     :param fill_include_names: list of names to include in fill_values (default=None selects all names)
     :param fill_exclude_names: list of names to exlude from fill_values (applied after ``fill_include_names``)
-    :param Reader: Reader class (DEPRECATED) (default=``ascii.Basic``)
+    :param Reader: Reader class (DEPRECATED) (default= :class:`Basic``)
```
Comment (embray) [inline astropy/io/ascii/ui.py]: There's an extra backtick at the end of "Basic" here.

File: astropy/io/ascii/ui.py [hunk 2]
```diff
@@ -268,17 +268,17 @@ def write(table, output=None,  format=None, Writer=None, **kwargs):
...
     :param quotechar: one-character string to quote fields containing special characters
     :param formats: dict of format specifiers or formatting functions
-    :param strip_whitespace: strip surrounding whitespace from column values (default=True)
+    :param strip_whitespace: strip surrounding whitespace from column values (default= ``True``)
     :param names: list of names corresponding to each data column
-    :param include_names: list of names to include in output (default=None selects all names)
+    :param include_names: list of names to include in output (default= ``None`` selects all names)
     :param exclude_names: list of names to exlude from output (applied after ``include_names``)
-    :param Writer: Writer class (DEPRECATED) (default=``ascii.Basic``)
+    :param Writer: Writer class (DEPRECATED) (default= :class:`Basic``)
```
Comment (embray) [inline astropy/io/ascii/ui.py]: Here as well.

**Comment by astrofrog** [comment]
@hamogu - can you remove ``[skip ci]`` from the last commit message? We should check no Sphinx warnings are raised with these changes.

**Comment by hamogu** [comment]
Upps, I just noticed that I apparently selected to 
hamogu wants to merge 2 commits into astropy:v0.4.x
That's of course wrong - this should go into master. Is there a way to change that after the fact or do I need to open a new PR?

**Comment by astrofrog** [comment]
@hamogu - I don't think there is a way to change it, you'll need to open a new PR

**Pull request closed without merging**